src/marqo/tensor_search/tensor_search.py

```diff
@@ -2648,3 +2691,33 @@ def delete_documents(config: Config, index_name: str, doc_ids: List[str]):
             document_ids=doc_ids,
         )
     )
+
+def normalize_vector(embeddings: Union[List[List[float]], ndarray]) -> List[List[float]]:
```

Comment:
nit: there might be a need in the future for us to accept `List[float]` as a parameter here as well.

Reply:
added